bin/mc

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+import argparse
+import subprocess
+from subprocess import check_output, Popen, PIPE, STDOUT
+
+
+def get_minikube_ip():
+    """
+    Get the minikube ip
+    """
+    # XXX: Sudo is needed on Linux. What about OSX and Win?
```

Comment:
Neither do I need it on my install. This should definitely be a configuration option residing in `$HOME/.mwc-rc` or similar.